_redirects

```diff
@@ -87,7 +87,7 @@
 /docs/concepts/cluster-administration/authenticate-across-clusters-kubeconfig   /docs/tasks/access-application-cluster/authenticate-across-clusters-kubeconfig/ 301
 /docs/concepts/cluster-administration/cluster-management/     /docs/tasks/administer-cluster/cluster-management/ 301
 /docs/concepts/cluster-administration/configure-etcd/     /docs/tasks/administer-cluster/configure-upgrade-etcd/ 301
-/docs/concepts/cluster-administration/device-plugins/     /docs/concepts/cluster-administration/network-plugins/ 301
+/docs/concepts/cluster-administration/device-plugins/     /docs/concepts/cluster-administration/device-plugins/ 301
 /docs/concepts/cluster-administration/etcd-upgrade/     /docs/tasks/administer-cluster/configure-upgrade-etcd/ 301
```

Comment:
I see that it was a mistake to redirect device-plugins to network-plugins. But I think the fix is to just delete the redirect all together.

Reply:
Done.